tests/model/test_model.py

```diff
@@ -1392,12 +1392,14 @@ def test__handle_typing_event(self, mocker, model,
     @pytest.mark.parametrize('event, final_muted_streams, ', [
         (
             {'property': 'in_home_view',
+             'op': 'update',
              'stream_id': 19,
              'value': True},
             {15}
         ),
         (
             {'property': 'in_home_view',
+             'op': 'update',
```

Comment:
Making this change seems like a good opportunity to also refactor the code itself to prepare for the next commit, ie. to join the if/elif sequence incorporate the `update`?

Reply:
I'm assuming you mean using `update` for identifying mute events?